doc/rst/developer/bestPractices/DCO.rst

```diff
@@ -0,0 +1,131 @@
+Getting started with Chapel and the Developer Certificate of Origin (DCO)
```

Comment:
I wonder if this document should be part of ContributorInfo to reduce the number of pages one has to visit to get started contributing to Chapel.

Reply:
I have the opposite reaction:  ContributorInfo.rst is currently a pretty massive and sprawling file, particularly given that it isn't rendered online anywhere to support easy hyperlinks.  For that reason, I think it'd work better as a series of files where distinct ones focus on distinct topics such that someone who's already familiar with (say) git and how to clone a repository could more easily skip past the introductory topics they already know and focus on things specific to our project.  In that light, I made DCO.rst a separate file so that (a) we could refer to it from multiple places / emails / gitter conversations (which I find more awkward to do for a specific section of a non-sphinxed .rst file), (b) someone who came to the project who was already familiar with DCOs could feel the need to skip right past it by not bothering to read the file.

Reply:
> I think it'd work better as a series of files where distinct ones focus on distinct topics such that someone who's already familiar with (say) git and how to clone a repository could more easily skip past the introductory topics they already know and focus on things specific to our project.

I find that model appealing. I do like that ordered progression of ContributorInfo.rst, so I hope we can preserve that even if it is split up across multiple pages. 

Might be worth adding a minimal issue or TODO comment in ContributorInfo.rst to document our desire to move away from a monolithic document, so it's clearer to others making changes to these docs in the future (unless you planned on knocking this out pretty soon).

Reply:
> I do like that ordered progression of ContributorInfo.rst, so I hope we can preserve that even if it is split up across multiple pages.

Definitely.  None of this is intended to throw the current flow of ContributorInfo.rst under the bus.  However, it's currently somewhat of a behemoth (in the sense of "I can't mentally keep track what all it does or doesn't cover") while also being a little confusingly non-linear at times.  For example, a bunch of basic "how to use git" information is contained within this file, yet somewhere in the middle, neither at the point when that step occurs nor in an obvious appendix.  Whereas if that information were in a distinct `git-basics.rst` file, it'd more obviously be an optional side-step that you could take if you didn't know git well, but could skip over if you were an expert (while also being easier to find / refer to if you were just browsing the contents of the bestPractices directory).

Reply:
That makes a lot of sense. I find that model even more appealing now. :)